google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,162 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.devtools.clouddebugger.v2
```

Comment:
To confirm, we want to change the API name (`{api}`) from `clouddebugger` to `debugger`, right?

Do we want that to happen upstream, or do we want our tools to notice the prefix in the name and strip it?